feat(registrar): edit titles in Submitted applications tab

xflord opened the pull request:
* edited the title on the Submitted applications page to reflect the current filter used

HejdaJakub reviewed (changes requested):
- Don't forget to handle also Czech translation - `PerunRegistrarTranslaion_cs.properties`. There is an old property `submittedTitle` (delete it please) and there are missing new properties. 
- For me, there is better readability with basic labels like "Open applications" and "Closed applications" (potentially registrations). There is a little mess in the naming of states cause in the new GUI is a state `Submitted` and in the registrar, the same state is `Waiting for approval`. For this reason, something like `Open submitted registrations` makes me a little confused.

xflord commented:
> * Don't forget to handle also Czech translation - `PerunRegistrarTranslaion_cs.properties`. There is an old property `submittedTitle` (delete it please) and there are missing new properties.
> 
>     * For me, there is better readability with basic labels like "Open applications" and "Closed applications" (potentially registrations). There is a little mess in the naming of states cause in the new GUI is a state `Submitted` and in the registrar, the same state is `Waiting for approval`. For this reason, something like `Open submitted registrations` makes me a little confused.

Changed the titles to be the same as the options in the filter ("Open applications"/"All applications"), updated the file for the Czech translation

HejdaJakub commented on perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/ExceptionResolverImpl.java:
```diff
@@ -120,7 +119,7 @@ private void resolve() {
 
 			setInfo(trans.alreadySubmitted(getBeanName()),
-					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.submittedTitle()));
+					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.openSubmittedTitle()));
```
Label `openSubmittedTitle()` doesn't exist now, but I think that `myApplications()` would be sufficient here since the same label is also presented in the header menu.
```suggestion
					text + trans.visitSubmitted(Window.Location.getHref().split("#")[0], application.getId(), trans.myApplications()));
```

HejdaJakub reviewed (changes requested):
The last small suggestion (it also fixes the build).
